### PR TITLE
AR customization example

### PR DESCRIPTION
**Pull request by chezzdev**
Checks:

* [ ] Update changelog
* [x] Rebase to `master` branch
* [x] Assign reviewers

Linked issues:
https://github.com/mapbox/mapbox-vision/issues/2217

### CODE REVIEW

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * The tutorial is based on "ARNavigation" example which shows how to get the route and set up Vision AR session.
+ */
+class ARCustomizationViewController: ARNavigationViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Adjust AR rendering quality
+        visionARViewController.set(arQuality: 0.8)
+
+        // Make sure lane is visible
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ARCustomizationViewController.swift:15]: nit: 
```suggestion
        // Enable lane rendering
```
?

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        visionARViewController.isLaneVisible = true
+        // Set lane length in meters
+        visionARManager.set(laneLength: 40)
+
+        // Create an instance of LaneVisualParams
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ARCustomizationViewController.swift:20]: nit:

```suggestion
        // Create an instance of `LaneVisualParams`
```
to reflect the comments about the fence below

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        // Once `fenceVisualParams` is configured, set it to the view controller
+        visionARViewController.set(fenceVisualParams: fenceVisualParams)
+    }
+
+    private func restoreDefaultValues() {
+        // Set the new lane visual params without changes
+        visionARViewController.set(laneVisualParams: LaneVisualParams())
+        // Set the new fence visual params without changes
+        visionARViewController.set(fenceVisualParams: FenceVisualParams())
+        // Make fence invisible
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ARCustomizationViewController.swift:59]: nit:
```suggestion
        // Disable fence rendering
```
to mirror comments about enabling rendering
?

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        visionARViewController.set(arQuality: 0.8)
+
+        // Make sure lane is visible
+        visionARViewController.isLaneVisible = true
+        // Set lane length in meters
+        visionARManager.set(laneLength: 40)
+
+        // Create an instance of LaneVisualParams
+        let laneVisualParams = LaneVisualParams()
+        // Set lane color
```
Comment (dersim-davaod) [inline Examples/VisionExamples/Examples/ARCustomizationViewController.swift:22]: nit:

```suggestion
        // Set the color of the lane
```

to mirror the comments about the fence below.

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift (hunk 1 above)
Comment (chezzdev) [reply Examples/VisionExamples/Examples/ARCustomizationViewController.swift:15]: Well, it's enabled by default. WDYT about 
```suggestion
        // Make sure lane is visible (default behavior).
```

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift (hunk 4 above)
Comment (chezzdev) [reply Examples/VisionExamples/Examples/ARCustomizationViewController.swift:22]: Let's mirror lane style in fence comments.

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift (hunk 3 above)
Comment (chezzdev) [reply Examples/VisionExamples/Examples/ARCustomizationViewController.swift:59]: Agree.

File: Examples/VisionExamples/Examples/ARCustomizationViewController.swift (hunk 2 above)
Comment (chezzdev) [reply Examples/VisionExamples/Examples/ARCustomizationViewController.swift:20]: Agree.

**Pull request merged**